spec/lang/step/expressions.md

```diff
@@ -45,6 +45,15 @@ impl<M: Memory> Machine<M> {
                     provenance: None,
                 }.widen(None))
             },
+            Constant::VTablePointer(vtable_name) => {
+                let mut vtables = self.vtable_allocs.iter().filter(|(_, name)| *name == vtable_name);
+                // we could also pick a random one?
+                let Some((ptr, _)) = vtables.next() else {
+                    panic!("constant to unallocated vtable");
```

Comment:
Does WF actually ensure that this panic cannot happen? I don't see it ensuring that.

Reply:
Well, it ensures `prog.vtables.contains_key(vtable_name)`, and here I assume `Machine` actually allocates every vtable in the program.